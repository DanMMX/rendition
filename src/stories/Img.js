import * as React from 'react'
import { storiesOf } from '@storybook/react'
import styled from 'styled-components'
import { Img } from '../'
import * as Logo from './assets/etcher.svg'

const Container = styled.div`
  margin: 30px;
  background: ${props => props.theme.colors.gray.dark};
`

storiesOf('Img', module)
  .addDecorator(story => <Container>{story()}</Container>)
  .addWithInfo('Standard', () => {
    return <Img p={5} src={Logo} />
  })
