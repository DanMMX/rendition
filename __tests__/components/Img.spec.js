import React from 'react';
import renderer from 'react-test-renderer';
import Provider from '../../src/components/Provider'
import Img from '../../src/components/Img'

test('Img renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Img />
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
