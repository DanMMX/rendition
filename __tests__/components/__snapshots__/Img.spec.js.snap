// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Img renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <img
    className="sc-bwzfXH fHoJvN"
  />
</div>
`;
